gensim/models/phrases.py

```diff
@@ -210,10 +210,12 @@ def load(cls, *args, **kwargs):
         # update older models
         # if value in phrasegrams dict is a tuple, load only the scores.
         try:
-            if isinstance(list(model.__dict__['phrasegrams'].values())[0], tuple):
-                model.__dict__['phrasegrams'].update((k, v[1]) for k, v in model.__dict__['phrasegrams'].items())
+            for components, scores in model.__dict__['phrasegrams'].items():
```

Comment:
Does this really work? It mutates the collection it's iterating over, which is usually a bad idea.

As mentioned previously, I'd make a copy of the original keys (not items) and iterate over that, while mutating the original (large) dict.